lib/vsdata.cpp

```diff
@@ -73,14 +75,20 @@ void VisualizationSceneScalarData::DoAutoscaleValue(bool prepare)
    }
 }
 
-void VisualizationSceneScalarData::Cone(gl3::GlBuilder& builder,
-                                        glm::mat4 xfrm)
+template<typename T>
+static std::array<float, 3> ToVec3(const T* vec)
+{
+   return { (float) vec[0], (float) vec[1], (float) vec[2] };
+}
+
+void VisualizationSceneScalarData::Cone(gl3::GlDrawable& buf,
+                                        glm::mat4 xfrm,
+                                        double cval)
 {
    const int n = 8;
```

Comment:
(minor/question) do you know if the compiler will do the same stuff with `const int n = 8` as `constexpr int n = 8`? should it change? (along with the other `const <numeric type>`)

Reply:
If I had to guess, they'd probably be equivalently optimizable in this case.

Reply:
Yes, I checked, at least with intel the output is the same in compiler explorer